test/LeanCode.Tests.csproj

```diff
@@ -16,9 +18,19 @@
       <TestProjectsAssemblies Include="@(_ResolvedProjectReferencePaths)" Condition="$([System.String]::new('%(FullPath)').Contains('test')) And '%(TargetFrameworks)' == 'net5.0'" />
     </ItemGroup>
 
+    <exec WorkingDirectory="$(TargetDir)" Command="dotnet vstest @(TestProjectsAssemblies -> '&quot;%(FullPath)&quot;', ' ')" />
+  </Target>
+
+  <Target Name="RunTestsWithCoverage" DependsOnTargets="Build">
     <exec
-      Command="dotnet vstest @(TestProjectsAssemblies -> '&quot;%(FullPath)&quot;', ' ') --logger:&quot;trx;LogFileName=$(LogFileName)&quot; --Blame"
-      WorkingDirectory="$(TargetDir)" />
+      ContinueOnError="ErrorAndContinue"
+      WorkingDirectory="$(TargetDir)"
+      Command="dotnet vstest \
+        '%(_ResolvedProjectReferencePaths.FullPath)' \
+        --logger:&quot;trx&quot; \
```

Comment:
Are quotes actually needed around string literals? Assuming this will be executed by native OS shell, `bash` and similar shells will just consume them and do nothing, making them unnecessary if that's the case here.